prow/plugins/label/label.go

```diff
@@ -188,7 +188,17 @@ func handle(gc githubClient, log *logrus.Entry, additionalLabels []string, e *gi
 
 	if len(nonexistent) > 0 {
 		log.Infof("Nonexistent labels: %v", nonexistent)
-		msg := fmt.Sprintf("The label(s) `%s` cannot be applied. These labels are supported: `%s`", strings.Join(nonexistent, ", "), strings.Join(additionalLabels, ", "))
+
+		// Check what additional labels are available on this repo
```

Comment:
If you make the GitHub `AddLabel` API call and it does not exist on the repo yet, it gets created. This is not technically correct and we should continue to tell the user  what labels the `/label` command is configured to allow.

Reply:
@stevekuznetsov thanks for the feedback! Based on the behavior I have observed, if you attempt to add a label that does not currently exist on the repo to an issue, it is not currently being added to the repo / issue. So we will continue to see the behavior that is in the issue referenced above. However, if this is the desired state, I would be happy to close this PR.

Reply:
I still think that the correct and appropriate fix for the issue is to only respond to a user's request with the message, as written, when they use `/label` with an unknown label.

Reply:
@stevekuznetsov so that would be the existing behavior, correct?

Reply:
Additionally, the fact that the message contains a snippet telling the user what they tried to do that was invalid (Liz, attempting to add the `priority/` label, Dan, the `sig/provider-azure` label) shows that something is parsing their messages to the bot incorrectly, since they are _not_ explicitly asking for those labels (and `priority/` is not a label at all). So to fix this, I would do one of the following:

 - find the offending PR that broke this and revert it
 - fix the logic in the `label` plugin to correctly detect when someone is using `/label` versus the other commands the plugin supports
 - fix the logic in the `label` plugin to correctly identify _which_ label is being requested by the user

Reply:
Apologies for my confusion, I think I have a good grasp now. I will let you know when updated!